src/generic/clipboard/paste-component/messages.js

```diff
@@ -0,0 +1,14 @@
+import { defineMessages } from '@edx/frontend-platform/i18n';
+
+const messages = defineMessages({
+  popoverContentText: {
+    id: 'course-authoring.generic.paste-component.popover.content.text',
+    defaultMessage: 'From:',
```

Comment:
Can we get descriptions in this file, please?

Reply:
Descriptions were added. Please recheck and LGTM.